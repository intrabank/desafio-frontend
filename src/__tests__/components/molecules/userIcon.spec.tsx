import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import UserIcon from '@/components/molecules/userIcon';

describe('UserIcon', () => {
	it('should render user icon component', () => {
		const tree = renderer.create(<UserIcon />).toJSON();
		expect(tree).toMatchSnapshot();
	});

	it('should render user initials ', () => {
		render(<UserIcon inititals='JP' />);

		const initials = screen.getByText('JP');
		expect(initials).toBeInTheDocument();
	});
});
