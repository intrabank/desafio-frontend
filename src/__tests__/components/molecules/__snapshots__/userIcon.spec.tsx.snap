// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserIcon should render user icon component 1`] = `
<div
  className="c-fBbncp"
/>
`;
